integrations/slack/src/handler.ts

```diff
@@ -33,7 +34,9 @@ export const handler: botpress.IntegrationProps['handler'] = async ({ req, ctx,
     const actionValue = await respondInteractive(body)
 
     if (body.type !== 'block_actions') {
-      throw Error(`Interaction type ${body.type} is not supported yet`)
+      const errMessage = `Interaction type ${body.type} is not supported yet`
+      logger.forBot().error(errMessage)
+      throw Error(errMessage)
```

Comment:
I think we can remove this line and just add an early return here as logs sent with `logger.forBot()` will also appear in the integration logs for the integration builder.